### ISSUE TITLE
scare quotes in title

### ISSUE THREAD

**Issue by martinthomson** [ietf-ribose/bibxml-service#189]
Are the quotes in `"IETF BibXML service"` intended to convey something?  Can they be removed?

**Comment by kesara** [comment]
This was a configuration option and I've removed the quotes.
Probably https://github.com/ietf-ribose/bibxml-service/blob/main/docs/howto/run-in-production.rst should be updated to reflect that quotes are passed in.

**Comment by strogonoff** [comment]
@kesara Indeed, Compose’s documentation [agrees with you](https://docs.docker.com/compose/env-file/#syntax-rules):

> There is no special handling of quotation marks. This means that they are part of the VAL.

[`.env` syntax in documentation has been corrected](https://dev.bibxml.org/static/docs/howto/run-in-production.html). I’m not sure how in my setup quotes ended up getting stripped (I used quotes, as in old samples in the docs).
